types/enzyme-adapter-react-15/enzyme-adapter-react-15-tests.ts

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import * as Adapter from 'enzyme-adapter-react-15';
```

Comment:
Could you use `import Adapter = require("enzyme-adapter-react-15")`?
An actual es6 namespace import would not be callable as done below. The adapter is, however, a commonjs file which does export a class as its entire export namespace, so the compromise is to use an import style which indicates that.